thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/template.hbs

```diff
@@ -66,22 +66,18 @@
         </div>
 
         <div class="rootcause-anomaly__stats-wrapper">
-          <div class="rootcause-anomaly__stats">
-            <label class="rootcause-anomaly__stats-label">Duration</label>
-            <span class="rootcause-anomaly__stats-value">{{duration}}</span>
-          </div>
-          <div class="rootcause-anomaly__stats">
-            <label class="rootcause-anomaly__stats-label">Current Avg</label>
-            <span class="rootcause-anomaly__stats-value">{{current}}</span>
-          </div>
-          <div class="rootcause-anomaly__stats">
-            <label class="rootcause-anomaly__stats-label">Baseline Avg</label>
-            <span class="rootcause-anomaly__stats-value">{{baseline}}</span>
-          </div>
-          <div class="rootcause-anomaly__stats">
-            <label class="rootcause-anomaly__stats-label">Change</label>
-            <span class="rootcause-anomaly__stats-value rootcause-anomaly__stats-value--{{color-delta change}}">{{changeFormatted}}</span>
-          </div>
+          {{#each-in anomalyInfo as |offset value|}}
```

Comment:
afaik this does not necessarily iterate in order. you may get 'wo2w', 'current', 'wow4', ...

Reply:
Good point. I'll have this loop through an array of ordered keys to iterate over the `anomalyInfo` object.